integration/walletextension/wallet_extension_test.go

```diff
@@ -32,99 +31,82 @@ import (
 	dummyAccountAddr = "0x8D97689C9818892B700e27F316cc3E41e17fBeb9"
 
 	startPort    = 3000
 	httpProtocol = "http://"
 
-	reqJSONMethodChainID    = "eth_chainId"
-	reqJSONMethodGetBalance = "eth_getBalance"
-	reqJSONMethodCall       = "eth_call"
-	respJSONKeyResult       = "result"
-	reqJSONKeyTo            = "to"
-	reqJSONKeyFrom          = "from"
-	emptyResult             = "0x"
-	errInsecure             = "enclave could not respond securely to %s request because there is no viewing key for the account"
+	reqJSONMethodChainID = "eth_chainId"
+	reqJSONKeyTo         = "to"
+	reqJSONKeyFrom       = "from"
+	emptyResult          = "0x"
+	errInsecure          = "enclave could not respond securely to %s request because it does not have a viewing key for account"
+
+	networkStartPort = integration.StartPortWalletExtensionTest + 1
+	nodeRPCHTTPPort  = integration.StartPortWalletExtensionTest + 1 + network.DefaultHostRPCHTTPOffset
+	nodeRPCWSPort    = integration.StartPortWalletExtensionTest + 1 + network.DefaultHostRPCWSOffset
+)
+
+var (
+	walletExtensionAddr   = fmt.Sprintf("%s:%d", network.Localhost, integration.StartPortWalletExtensionTest)
+	walletExtensionConfig = walletextension.Config{
```

Comment:
I cleaned up the tests by defining the various ports and config object just once. This makes the set up in each test much easier to follow.